app/models/host/managed.rb

```diff
@@ -550,6 +549,17 @@ def self.count_habtm(association)
     association.camelize.constantize.find(counter.keys.compact).map {|i| {:label=>i.to_label, :data =>counter[i.id]}}
   end
 
+  def self.provision_methods
+    Hash[{
```

Comment:
I don't think converting the sorted array into a hash again will preserve the sort order, might be safer to do it in the UI.